xml/planning.xml

```diff
@@ -163,4 +163,61 @@
    For registering an already installed system, see <xref linkend="sec.yast-software.register"/>.
   </para>
  </sect1>
+ 
+ <sect1 xml:id="sec.planning.leanos">
+  <title>Changes in installation from &sls; version &product-ga;</title>
+  <para>
+   Starting with &productname; 15, all products are distributed using a single
+   set of installation media.
+  </para>
+  <para>
+   The products available on these unified media are as follows:
+  </para>
+  <variablelist>
+   <varlistentry>
+    <term>SLES</term>
+    <listitem>
+     <para>
+      &sls;
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>SLE_HPC</term>
+    <listitem>
+     <para>
+      &slehpc;
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>SLES_SAP</term>
+    <listitem>
+     <para>
+      &sles4sap;
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>SLED</term>
+    <listitem>
+     <para>
+      &sled;
+     </para>
+    </listitem>
+   </varlistentry>
+  </variablelist>
+  <para>
+   This is subject to the existing restriction that not all products are
+   supported on all architectures.
+  </para>
+  <para>
+   To install any of these products, it is only necessary to download the
+   <filename>&slea; &productnumber; Installer DVD 1</filename> ISO image. 
+  </para>
+  <para>If you want to install without network access, additionally download
+   the other two ISO images: <filename>&slea; &productnumber; Installer DVD 2
+   </filename> and <filename>&slea; &productnumber; Installer DVD 3</filename>. 
+  </para>
+ </sect1>
```

Comment:
What should also be mentioned (according to bsc#1123063) is the 'Packages' image (use the &allmodules; entity to make it include the correct product version number). The Packages image is needed if you want to install without access to SCC or a registration server (RMT).

Reply:
Good point. I found this: http://download.suse.de/install/SLE-15-Packages-GM/ (not for customer use, just as a reference).

It appears to confirm that there is 1 Packages ISO for each arch, *but* there seem to be 3 ISOs per arch: "SLE-15-Packages-x86_64-GM-DVD1.iso", "SLE-15-Packages-x86_64-GM-DVD2.iso", & ""SLE-15-Packages-x86_64-GM-DVD1.iso".

Reply:
Received clarification on Bugzilla from Kai Dupke and Vincent Mouttoussamy -- Packages 2 medium is just source code; never needed for installation. Packages 3 is for beta/internal use only. Clarified role of Packages 2 in docs, and removed mention of Packages 3.